.github/workflows/CI.yml

```diff
@@ -425,7 +425,7 @@ jobs:
     strategy:
       matrix:
         go-version: [ 1.13.x ]
-        platform: [ ubuntu-20.04 ] #[ ubuntu-20.04, macOS-11.0 ] # , windows-2019
+        platform: [ ubuntu-20.04 ] #, macOS-11.0, windows-2019 ]
```

Comment:
is windows-2019 disabled on purpose?

Reply:
Yes, as it was before this PR, I only restructured the list. 
Tests are currently failing on Windows, and on MacOS we don't have enough runners on GitHub, so the workflow often just stalls/times out.